backend/routes/rides.js

```diff
@@ -16,20 +17,29 @@ router.get('/', async function (req, res, next) {
     //TODO: validate login state - login user = given userid
     //TODO: search logic for begin and end date
     const query = JSON.parse(req.query['query']);
-    console.log(query);
+    if (DEBUG) {
+        console.log(query);
+    }
     const originCoords = query['originCoords'];
```

Comment:
I just realized a better way of doing this would be 
`    const {originCoords, destCoords, beginDate, endDate, distance, oranyfieldyouwant} = query;
`
And you won't need to comment things out.

Reply:
I like this, let me add it in